pyfstat/helper_functions.py

```diff
@@ -912,16 +913,30 @@ def get_predict_fstat_parameters_from_dict(signal_parameters):
         helper_functions.predict_fstat.
         This dictionary's keys must follow
         the PyFstat convention (e.g. F0 instead of Freq).
+    transientWindowType: str
+        Transient window type to store in the output dict.
+        Currently required because the typical input dicts
+        produced by various PyFstat functions
+        tend not to store this property.
+        If there is a key with this name already, its value will be overwritten.
 
     Returns
     -------
     predict_fstat_params: dict
         The dictionary of selected parameters.
     """
-    predict_fstat_params = {
-        key: signal_parameters[key]
-        for key in ["F0", "Alpha", "Delta", "h0", "cosi", "psi"]
+    required_keys = ["F0", "Alpha", "Delta", "h0", "cosi", "psi"]
+    transient_keys = {
+        "transientWindowType": "transientWindowType",
+        "transient_tstart": "transientStartTime",
+        "transient_duration": "transientTau",
```

Comment:
Interestingly enough, these keys don't match the ones in `BaseSearchClass.translate_keys_to_lal`.

Reply:
I noticed that too. Those are for CFSv2, this is for PFS, which don't agree with each other.